docs/openapi.json

```diff
@@ -647,17 +647,33 @@
               "application/json": [
                 {
                   "access_token": "Drmhze6EPcv0fN_81Bj-nA",
-                  "cpu_usage": "70536",
-                  "disk_usage": "408",
                   "email": "user@reana.info",
-                  "id": "00000000-0000-0000-0000-000000000000"
+                  "id": "00000000-0000-0000-0000-000000000000",
+                  "quota": {
+                    "cpu": {
+                      "limit": "200000",
+                      "usage": "70536"
```

Comment:
Shouldn't it be an int if it's type "number"?

Reply:
That's what I thought, but then I realised we won't be able to fit big integers into integers... so we are left with two options: `string` or `number` without format (representing "any number" see [here](https://swagger.io/docs/specification/data-models/data-types/#numbers)).